src/Files.App/UserControls/Widgets/RecentFilesWidget.xaml.cs

```diff
@@ -1,13 +1,13 @@
-// Copyright (c) 2024 Files Community
-// Licensed under the MIT License. See the LICENSE.
-
+using Microsoft.Extensions.Logging;
```

Comment:
```suggestion

```
Is there a need?

Reply:
No need for logging thanks for spotting that